common/src/main/java/net/dodogang/plume/cosmetic/Cosmetic.java

```diff
@@ -0,0 +1,18 @@
+package net.dodogang.plume.cosmetic;
+
+import net.minecraft.util.Identifier;
+
+public class Cosmetic {
+    public final Identifier id;
+    public final CosmeticSlot slot;
+
+    public Cosmetic(Identifier id, CosmeticSlot slot) {
+        this.id = id;
+        this.slot = slot;
+    }
+
+    @Override
+    public String toString() {
+        return "Cosmetic{" + "id=" + id + ", slot=" + slot + '}';
```

Comment:
Is this only for debugging, or will this be used as a way to send it to the vps? If so, it should probably in a json format so the equal signs should be colons.

Reply:
This is an auto generated toString method used for debugging, yeah
NBT and shizzle will come later for networking